src/javascript/app/Modules/Trading/Components/Form/TradeParams/Duration/simple-duration.jsx

```diff
@@ -1,6 +1,7 @@
 import { PropTypes as MobxPropTypes } from 'mobx-react';
 import PropTypes                      from 'prop-types';
 import React, { Fragment }            from 'react';
+import { connect }                    from 'Stores/connect';
```

Comment:
should be on line 8

Reply:
it gives error if i put on line 8, its fine on 7 👍

Reply:
LOL why?

Reply:
like usual `eslint import order ....` 😂

Reply:
are you sure? I don't get that

Reply:
![image](https://user-images.githubusercontent.com/13959654/58239880-7e4e7b00-7d7c-11e9-9bd9-079d06c1dd2c.png)